config/checkstyle_resources_checks.xml

```diff
@@ -25,7 +25,7 @@
     <property name="folderPattern"
              value="[\\/]src[\\/](it|test)[\\/]resources(-noncompilable?)[\\/]"/>
     <property name="fileNamePattern"
-             value="^(package-info.java)|(Input\w+\.java)|(.*\.properties)$"/>
+             value="^(package-info.java)|(Input\w+\.java)|(Expected\w+\.txt)|(.*\.properties)$"/>
```

Comment:
File name pattern changed but violation message wasn't.

Reply:
Thanks for pointing this out, done.